package.json

```diff
@@ -102,6 +110,8 @@
     "postversion": "git push && git push --tags",
     "test": "jest --config config/jest/jest.config.js",
     "test:travis": "jest --maxWorkers=4 --config config/jest/jest.config.js",
-    "test:watch": "jest --config config/jest/jest.config.js --watch --coverage=false"
+    "test:watch": "jest --config config/jest/jest.config.js --watch --coverage=false",
+    "storybook": "start-storybook -p 6006",
+    "build-storybook": "build-storybook"
```

Comment:
Please alphabetize. Also, we're using `:` as a convention, so this would be `build:storybook`.

Reply:
`build-storybook` was created by the install process
```npx -p @storybook/cli sb init```
I'm not really sure what it does yet.